pom.xml

```diff
@@ -50,12 +50,44 @@
             <artifactId>spring-cloud-core</artifactId>
             <version>${spring-cloud.version}</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.springframework/spring-core -->
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
+            <version>5.0.3.RELEASE</version>
```

Comment:
this can be marked with `scope=test`?

Reply:
Yep, can't have a dependency on Spring directly.